python/cudf/cudf/io/parquet.py

```diff
@@ -164,7 +154,7 @@ def write_to_dataset(
         )
 
     else:
-        filename = filename or uuid4().hex + ".parquet"
+        filename = filename or (uuid4().hex + ".parquet")
```

Comment:
I see the default filename generated via uuid4 in a couple of places, and I'd prefer to see this in a small helper function so that the name generation stays consistent if for instance we change to uuid1 or something else.